### DISCUSSION TITLE
How do I generate the value for the "git-tree" field in the version file?

### DISCUSSION THREAD

**Discussion by eladm-ultrawis** [microsoft/vcpkg#30864] [Q&A]
In my local environment, I created a news port for a header only library (cuda-api-wrappers):

1. in ports/cuda-api-wrappers I added the vcpkg.json and portfile.cmake
2. in versions/baseline.json I added an entry:
```
	"cuda-api-wrappers" : {
	  "baseline": "0.6.2",
	  "port-version": 0
	}

```
3. now I wish to add versions/c-/cuda-api-wrappers.json, but I don't know how to fill the "git-tree" entry.
```
{
  "versions": [
    {
      "git-tree": "WHAT IS THIS AND WHERE DOES IT COME FROM????",
      "version-string": "0.6.2",
      "port-version": 0
    }
  ]
}
```

Can you help me out?

Thanks.

**Comment by autoantwort** [comment]
You can find an example here: https://learn.microsoft.com/en-us/vcpkg/maintainers/registries
